client/packages/common/src/ui/components/navigation/Tabs/DetailTabs.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, ReactNode, useState } from 'react';
+import { TabContext } from '@mui/lab';
+import { Box } from '@mui/material';
+import { LocaleKey, useTranslation } from '@common/intl';
+import { AppBarTabsPortal } from '../../portals';
+import { DetailTab } from './DetailTab';
+import { ShortTabList, Tab } from './Tabs';
+
+export type TabDefinition = {
```

Comment:
started with the approach of using `children` and converting to an array because I liked the declarative usage in the calling component. That did mean having a separate prop which passed in an array of values & labels and I thought the pain of separation was not balanced by the beauty of declarative syntax. So here we are. Reduced the label and value down to a single property, though some might argue for separation

Reply:
Like how it is done now!